web/src/components/application-detail-page/application-state-view/live-state-view/graph-view/resource-connector/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from "react";
+import { theme } from "~/theme";
+
+type Props = {
+  top: number;
+  left: number;
+  width: number;
+  height: number;
+  points: string;
+};
+
+const STROKE_WIDTH = 2;
+
+const ResourceConnector: FC<Props> = ({
```

Comment:
[nit] How about `ResourceEdge`? (also dir name) It becomes a pair with `ResourceNode` to draw g graph.